Content.Shared/Wires/SharedWiresSystem.cs

```diff
@@ -15,25 +16,41 @@ public override void Initialize()
 
     private void OnExamine(EntityUid uid, WiresPanelComponent component, ExaminedEvent args)
     {
-        args.PushMarkup(Loc.GetString(component.Open
-            ? "wires-panel-component-on-examine-open"
-            : "wires-panel-component-on-examine-closed"));
+        if (component.Open == false)
+        {
+            args.PushMarkup(Loc.GetString("wires-panel-component-on-examine-closed"));
+        }
+
+        else
+        {
+            args.PushMarkup(Loc.GetString("wires-panel-component-on-examine-open"));
+
+            if (component?.WiresPanelSecurityExamination != null)
+            {
+                args.PushMarkup(Loc.GetString(component.WiresPanelSecurityExamination));
+            }
+        }
     }
 
     private void OnGetState(EntityUid uid, WiresPanelComponent component, ref ComponentGetState args)
```

Comment:
Should use `AutoGenerateComponentState` on component

Reply:
So, I add the [AutoGenerateComponentState] attribute to WiresPanelComponent, add the 'partial' keyword, add the [AutoNetworkedField] attribute to the four fields, then delete the WiresPanelComponentState class? Anything else?

Reply:
Have to delete the OnGetState and OnHandleState in SharedWiresSystem and the corresponding SubscribeLocalEvents as well then?

I'm just not sure what [AutoGenerateComponentState] handles exactly

Reply:
https://docs.spacestation14.io/en/super-simple-14/networking-and-you

Reply:
Awesome, cheers